spring-cloud-gcp-logging/src/main/java/org/springframework/cloud/gcp/logging/LoggingWebMvcConfigurer.java

```diff
@@ -25,18 +27,22 @@
  * MVC Adapter that adds the {@link TraceIdLoggingWebMvcInterceptor}
  *
  * @author Mike Eltsufin
+ * @author Chengyuan Zhao
  */
 @Configuration
 public class LoggingWebMvcConfigurer extends WebMvcConfigurerAdapter {
 
 	private final TraceIdLoggingWebMvcInterceptor interceptor;
 
-	public LoggingWebMvcConfigurer(@Autowired(required = false) TraceIdLoggingWebMvcInterceptor interceptor) {
+	public LoggingWebMvcConfigurer(
+			@Autowired(required = false) TraceIdLoggingWebMvcInterceptor interceptor) {
 		if (interceptor != null) {
 			this.interceptor = interceptor;
 		}
 		else {
-			this.interceptor = new TraceIdLoggingWebMvcInterceptor();
+			this.interceptor = new TraceIdLoggingWebMvcInterceptor(
+					new CompositeTraceIdExtractor(ImmutableList.of(
```

Comment:
Somehow we try to avoid in Spring ecosystem Google Guava: too much breaking changes in the point releases and too much API copying names from other places what is misleading to the wrong imports during development. So, I would insist to come back to the vararg in the `CompositeTraceIdExtractor` and don't make end-users to import Guava code at all.